src/big-query-functions/big-query-service-class.ts

```diff
@@ -0,0 +1,11 @@
+import createTable from "./create-table";
+import deleteTable from "./delete-table";
+import insertRowsAsStream from "./insert-rows";
+import tableExists from "./table-exists";
+
+export default class BQ {
+    static createTable = createTable;
```

Comment:
Don't import the functions here. Move the functions here only and drop those files.